fapi-client/src/main/scala/com/gu/facia/api/models/collectionconfig.scala

```diff
@@ -1,12 +1,13 @@
 package com.gu.facia.api.models
 
-import com.gu.facia.client.models.{Backfill, CollectionConfigJson}
+import com.gu.facia.client.models.{Backfill, CollectionConfigJson, Metadata}
 
 case class Groups(groups: List[String])
 
 case class CollectionConfig(
     displayName: Option[String],
     backfill: Option[Backfill],
+    metadata: Option[List[Metadata]],
```

Comment:
A collection can have many of them.  They appear as `tags` in the fronts tool.
Eg:
![image](https://cloud.githubusercontent.com/assets/1722550/15295664/7b8af090-1b89-11e6-8bf3-059385a7e274.png)